<!doctype html>
<html>
<head>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width">


<title>Masayuki USHIO</title>


<link href="style.css" rel="stylesheet" type="text/css">



</head>

<body>
<!-- 2014.10.10 Ushio added-->
<div id="site-box">

<!-- ///////////////////////////////////////////////// header -->
<p class="switch">English / <a href="index_jp.html">Japanese</a></p>

<header>
<h1><a href="https://ong8181.github.io/"><span class="blue">Masayuki Ushio</span>'s Website</a></h1>
</header>


<!-- ///////////////////////////////////////////////// navigation -->
<nav>
<ul>
<li><a href="index.html">Home</a></li>
<li><a href="profile.html">Profile</a></li>
<li><a href="research.html">Research</a></li>
<li><a href="publications.html">Publications</a></li>
<li><a href="gallery.html">Gallery</a></li>
<li class="rline"><a href="link.html">Link</a></li>
</ul>
</nav>

<!-- ///////////////////////////////////////////////// section -->
<section>
<p></p>

<h2>Research topics</h2>

<h3>1. Nonlinear time-series analysis (2014-)</h3>
<p>In recent decades, nonlinear methods that acknowledge state dependence have been developed based on state space reconstruction (SSR). These methods do not assume any set of equations governing the system but recover the dynamics from time series data, thus called Empirical Dynamic Modeling (EDM). I am curerntly analyzing ecological time series using EDM  to forecast and understand state-dependent behaviour of complex ecological dynamics.</p><br>

<p><u>Related papers</u></p>
<p>Ushio 2022 <i>Proceedings of the Royal Society B</i> 289:20212690</p>
<p>Kitayama et al. 2021 <i>Journal of Ecology</i> 109 (2):727-736</p>
<p>Ushio et al. 2020 <i>Ecological Research</i> 35 (1):17-29</p>
<p>Ushio et al. 2018 <i>Nature</i> 554 (7692): 360-363</p>
<p>Chang et al. 2017 <i>Ecological Research</i> 32 (6): 785-796</p>

<h3>2. Developments and applications of environmental DNA techniques (2015-)</h3>
<p>I have been developing methods to monitor ecosystem dynamics using DNAs derived from environmental samples (e.g., soil and water) (i.e., environmantel DNA; eDNA).</p><br>

<p>Related papers</p>
<p>Ushio 2022 <i>Proceedings of the Royal Society B</i> 289:20212690</p>
<p>Yonezawa et al. 2020 <i>Consevation Genetics</i> 21 (6): 1078-1084</p>
<p>Ushio 2019 <i>Methods in Ecology and Evolution</i> 10 (8):142-1156</p>
<p>Ushio et al. 2018 <i>Metabarcoding & Metagenomics</i> 2: e23297</p>
<p>Ushio et al. 2017 <i>Molecular Ecology Resouces</i> 17: e63-e75</p>
<p>Ishige et al. 2017 <i>Biological Consevation</i> 210: 281-285</p>


<h3>3. Plant-soil feedbacks in a forest ecosystem (2005-)</h3>
<p>I have been working on plant-soil feedbacks in a forest  ecosystem, mainly in a forest in Malaysian Borneo. I am focusing on the relationship among plant secondary metabolites, soil microbial community, decomposition process and ecosystem dynamics. These works are still ongoing.</p><br>

<p><u>Related papers</u></p>
<p>Ushio et al. 2017 <i>Ecological Monographs</i> 87 (1): 105-129</p>
<p>Ushio et al. 2013 <i>Plant and Soil</i> 365 (1-2): 157-170</p>
<p>Ushio et al. 2010 <i>Soil Biology & Biochemistry</i> 42 (9): 1588-1595</p>
<p>Ushio et al. 2008 <i>Soil Biology & Biochemistry</i> 40 (10): 2699-2702</p>

<h3>4. Lab experiments on soil microbial ecology (2010-2014)</h3>
<p>I did mesocosm experiments to study the relationship between diversity and function of soil microbial community. I also tested the performance of several methodologies commonly used in soil microbial ecology. Lipid biomarker method, CARD-FISH and high-throughput sequencer were used in these studies.</p><br>
<p><u>Related papers</u></p>
<p>Ushio et al. 2014 <i>Soil Biology & Biochemistry</i> 76: 53-56</p>
<p>Ushio et al. 2013 <i>Soil Biology & Biochemistry</i> 64: 147-154</p>
<p>Ushio et al. 2013 <i>PLos ONE</i> 8 (11) e80320</p>

<h3>5. Theoretical analysis of plant-soil feedbacks (2008-2011)</h3>
<p>The consequences of plant-soil feedbacks for nutrient cycling and plant species coexistence were studied using mathematical models.</p><br>
<p><u>Related papers</u></p>
<p>Miki et al. 2010 <i>PNAS</i> 107 (32): 14251-14256</p>
<p>Ushio et al. 2009 <i>Microbes and Environment</i> 24 (2): 180-187</p>

<h3>6. Others (collaborative works etc.)</h3>
<p>I am also working on several different topics; e.g., environmental DNA, microbial fingerprint, theoretical formulations of prey-predator relationships etc...</p><br>

<p><u>Related papers</u></p>
<p>Takeuchi et al. 2011 <i>PLoS ONE</i> 6 (9) e25144</p>
<p>Nakazawa et al. 2013 <i>Biology Letters</i> 9 (3) no.20121193</p>
<p>Ushio et al. 2015 <i>Scientific Reports</i> 8: 8695</p>
<p>Ushio et al. 2017 <i>Molecular Ecology Resources</i> 17 (6): e63-e75</p>
<p>Ushio et al. 2018 <i>Metabarcoding & Metagenomics</i> 2: e23297</p>


<h2>Field work experience</h2>
<li>Mt. Kinabalu, Borneo, Malaysia (2005-2016)</li>
<li>UW-Madison Arboretum, Wisconsin, USA (2010)</li>
<li>Abisko, Sweden (2011)</li>
<li>South province in New Caledonia (2013)</li>
<li>Rice paddy field in Japan (2016-2019)</li>

<h2>Research funds</h2>
*Research projects as a principal investigator only
<li>2024.9 - 2027.8　RGC General Research Fund (HKD 1,323,889) "Establishment of an environmental DNA-based cetacean monitoring framework in Hong Kong waters"</li><br>

<li>2024.7 - 2027.6　Marine Ecology Enhancement Fund (HKD 273,600 for FY2024) "Developing a non-invasive method to estimate the age of Chinese White Dolphins and Finless Porpoises using the DNA methylation rate
"</li><br>

<li>(Completed) <font color="gray"> 2020.4 - 2024.3,  JSPS Grand-in-aid for Scientific Research (B) &quot;Development of a framework for controlling aquatic ecosystem dynamics using host-parasite systems as a model system&quot; JPY 13,300,000 (approx. USD 110,000)</font></li><br>

<li>(Completed) <font color="gray"> 2018.10 - 2023.9,  Kyoto University Hakubi Project &quot;Automated ecosystem monitoring system for forecasting ecological dynamics&quot; JPY 12,650,000 (approx. USD 101,000)</font></li><br>

<li>(Completed) <font color="gray"> 2016.10 - 2020.3,  Japan Science and Technology Agency, PRESTO &quot;Forecasting agricultural ecosystem dynamics using ecological networks&quot; JPY 40,000,000 (approx. USD 360,000)</li></font><br>

<li>(Completed) <font color="gray">  2014.4 - 2017.3,  JSPS Grand-in-aid for Young Scientists (B) &quot;Identification and quantification of microbial pathogens that drive plant-soil feedbacks in a forest ecosystem&quot; JPY 3,100,000 (approx. USD 31,000)</li></font><br>

<li>(Completed) <font color="gray"> 2012.4 - 2014.3, Grand-in-aid from Institution for Fermentation, Osaka &quot;Microbes on a pollen: their detection and its application&quot; JPY 3,000,000 (approx. USD 38,000)</li></font><br>

<li>(Completed) <font color="gray"> 2011.4 - 2014.3, JSPS Research Fellowship for Young Scientist PD (23-586) &quot;A new terrestrial ecosystem model incorporating microbial functional diversity and dispersal process&quot; JPY 2,400,000 (approx. USD 30,000)</li></font><br>

<li>(Completed) <font color="gray"> 2010.4 - 2010.9, JSPS Excellent Young Researcher Oversea Visit Program. JPY 1,060,840 (approx. USD 13,000)</li></font><br>

<li>(Completed) <font color="gray"> 2009.4 - 2011.3, JSPS Research Fellowship for Young Scientist DC2 (21-1526) &quot;Elucidation of the mechanism of forest ecosystem dynamics regulated by soil microbial communities: a model study in a tropical montane forest&quot; JPY 1,400,000 (approx. USD 17,000)</font></li>

<h2>Research skills</h2>
<ul>
<li>Soil pH, moisture, C and N content</li>
<li>Soil nutrient quantification (P, N)</li>
<li>Phenolic quantification (total phenolics and condensed tannins)</li>
<li>Soil respiration rate (LI6400, INNOVA)</li>
<li>Soil enzyme activity (<i>p</i>-nitrophenol substrate, MUB substrate)</li>
<li> Analysis of soil microbial community (lipid biomarker analysis, CARD-FISH)</li>
</ul><br>

<ul>
<li>Extraction and purification of condensed tannins from plant leaves</li>
<li>Nitrogen fixation activity (acethylene reduction assay)</li>
<li>Utilization ability of organic nitrogen by plant roots (stable isotope tracer)</li>
<li>Measurements of sapling growth rates in field</li>
<li>Evaluation of rice performance in field</li>
<li>Rice RNA extraction</li>
</ul><br>

<ul>
<li>Environmental DNA sampling (GF/F filter, Sterivex)</li>
<li>Extraction and PCR of eDNA (Fish, mammal, bird, invertebrate etc...)</li>
<li>Quantitative and multispecis monitoring of eDNA (Quantitative MiSeq)</li>
<li>Environmental DNA sequencing (IonPGM, MiSeq, iSeq, MinION)</li>
</ul><br>

<ul>
<li>Construction and analysis of mathematical model</li>
<li>Statistics by R and Python</li>
<li>C/C++ (beginner-level)</li>
<li>Sequence data analysis (Claident, QIIME, UPARSE, DADA2, phyloseq)</li>
<li>Nonlinear time-series analysis (Empirical Dynamic Modeling)</li>
</ul>

</section>


<footer>
<p>&copy; Masayuki USHIO</p>
</footer>

</div>
</body>
</html>
